rviz_default_plugins/src/rviz_default_plugins/displays/pointcloud/point_cloud2_display.cpp

```diff
@@ -179,13 +179,13 @@ void PointCloud2Display::update(float wall_dt, float ros_dt)
 
 void PointCloud2Display::reset()
 {
-  RTDClass::reset();
+  MFDClass::reset();
   point_cloud_common_->reset();
 }
 
 void PointCloud2Display::onDisable()
 {
-  RosTopicDisplay::onDisable();
+  MessageFilterDisplay::onDisable();
```

Comment:
```suggestion
  MFDClass::onDisable();
```

Or, TBH, I think it would be more clear to update all instances of `MFDClass::` to `MessageFilterDisplay::`. Perhaps out of scope for this PR though.

Reply:
I think we can do the conversion `MFDClass` -> `MessageFilterDisplay` in this PR itself. Should I do it?

Reply:
On second thought, I think we should keep using `MFDClass`. It looks like this was [added to avoid having to write the template type](https://github.com/ros2/rviz/blob/c05726637c6f91718d33e0bc9ee2717a688552b1/rviz_common/include/rviz_common/message_filter_display.hpp#L52-L54). It makes me wonder how this code is compiling to begin with.